Resources/Prototypes/Nyanotrasen/Entities/Clothing/Belt/belts.yml

```diff
@@ -34,4 +34,49 @@
   - type: Sprite
     sprite: Nyanotrasen/Clothing/Belt/Martial/black.rsi
   - type: Clothing
-    sprite: Nyanotrasen/Clothing/Belt/Martial/black.rsi
\ No newline at end of file
+    sprite: Nyanotrasen/Clothing/Belt/Martial/black.rsi
+
+- type: entity
+  parent: ClothingBeltStorageBase
+  id: ClothingBeltKatanaSheath
+  name: katana sheaths
+  description: An ornate belt designed to hold a pair of samurai's blades, or daisho.
+  components:
+  - type: Sprite
+    sprite: Nyanotrasen/Clothing/Belt/katanasheath.rsi
+    state: sheath
+  - type: Clothing
+    sprite: Nyanotrasen/Clothing/Belt/katanasheath.rsi
+  - type: Storage
+    capacity: 25
+    whitelist:
+      tags:
+        - Katana
+  - type: ItemMapper
+    mapLayers:
+      sheath-sabre:
+        whitelist:
+          tags:
+          - Katana
+  - type: Appearance
+  - type: Tag
+    tags:
+      - BeltSlotNotBelt
+
+- type: entity
+  parent: ClothingBeltStorageBase
+  id: ClothingBeltMilitaryWebbingCMO
```

Comment:
Not a fan of giving military hardware to roles that don't need it.

Reply:
As before, I used webbing instead of a belt mostly due to aesthetics. I also reduced the capacity of the belt per Rane's request. If you would like me to implement further changes, I can do so.